src/wp-content/themes/twentytwentyone/inc/back-compat.php

```diff
@@ -36,7 +36,7 @@ function twenty_twenty_one_switch_theme() {
  * @return void
  */
 function twenty_twenty_one_upgrade_notice() {
-	echo '<div class="error"><p>';
+	echo '<div class="notice notice-error"><p>';
```

Comment:
Don't change back-compat classes, as the context where these run may not include the necessary support.